apps/judicial-system/backend/src/app/modules/notification/notification.service.ts

```diff
@@ -474,6 +512,7 @@ export class NotificationService {
       const pdf = await getRequestPdfAsString(existingCase, intl.formatMessage)
 
       attachments = [
+        this.createICalAttachment(existingCase),
```

Comment:
You probably want to send the calendar invite even if the request PDF is not sent. Consider adding the calendar invite outside the if statement.